Applications/CLI/ogs.cpp

```diff
@@ -117,9 +137,22 @@ int main(int argc, char *argv[])
 
         BaseLib::ConfigTree::assertNoSwallowedErrors();
 
-        return solver_succeeded ? EXIT_SUCCESS : EXIT_FAILURE;
+        ogs_status = solver_succeeded ? EXIT_SUCCESS : EXIT_FAILURE;
     } catch (std::exception& e) {
         ERR(e.what());
-        return EXIT_FAILURE;
+        ogs_status = EXIT_FAILURE;
     }
+
+    {
+        auto const end_time_sys = std::chrono::system_clock::to_time_t(
+            std::chrono::system_clock::now());
+        char time_str[100];
+        if (std::strftime(time_str, sizeof(time_str), "%F %T %z",
+            std::localtime(&end_time_sys))) {
+	    INFO("OGS terminated on %s.", time_str);
+	}
```

Comment:
Here you could use the suggested general BaseLib function and avoid code duplication.